crates/http/src/error.rs

```diff
@@ -7,6 +7,9 @@ use axum::{
 pub enum Error {
     #[error("serialization error: {0}")]
     Serde(#[from] serde_json::Error),
+
+    #[error("foundry error")]
+    Foundry(String),
```

Comment:
I think you should be able to do

```suggestion
    Foundry(#[from] iron_forge::Error),
```

Reply:
I don't think I can because the `foundry_get_abi` function in the`iron-forge` crate returns a String on the `Result<Option<Abi>, String>` and not an Error

Reply:
ah. then that needs to be fixed on the other side. something for later then 👍🏼